src/cmd-init

```diff
@@ -24,12 +25,15 @@ Usage: coreos-assembler init --help
 
   If specified, SUBDIR is a subdirectory of the git repository that should
   contain manifest.yaml and image.yaml (or image.ks).
+
+  Use `--transient` for builds that will throw away all cached data on success/failure,
+  and should hence not invoke `fsync()` for example.
 EOF
 }
 
 # Call getopt to validate the provided input.
 rc=0
-options=$(getopt --options hfb:c: --longoptions help,force,branch:,commit: -- "$@") || rc=$?
+options=$(getopt --options hfb:c: --longoptions help,force,transient,branch:,commit: -- "$@") || rc=$?
```

Comment:
Missing the short option

```suggestion
options=$(getopt --options hftb:c: --longoptions help,force,transient,branch:,commit: -- "$@") || rc=$?
```

Reply:
Well, that was intentional...I think this isn't going to be common enough to warrant a short option.  Long options are good for things like this that end up in scripts mainly, aren't typed by hand frequently and should be clear and explicit.